### PR TITLE
Teams update

### PR DESCRIPTION
**Pull request by Mattsface**
### Why
As a developer, I want to be able to search multiple JSON keys when looking for a ID

### What
Added  params to get_team_id and a try except block incase of keyerror

### Tests
PyTest

### Risk and impact
- Minima

What is the impact if something does go wrong with this PR?

### CODE REVIEW

File: mlbstatsapi/mlb_api.py [hunk 1]
```diff
@@ -242,15 +247,17 @@ def get_team_id(self, team_name) -> List[int]:
...
+        mlb_data = self._mlb_adapter_v1.get(endpoint='teams', ep_params=params)
         team_ids = []
-
+    
         if 'teams' in mlb_data.data and mlb_data.data['teams']:
             for team in mlb_data.data['teams']:
-                if team['name'].lower() == team_name.lower():
-                    team_ids.append(team['id'])
-
+                try:
```
Comment (KCNilssen) [inline mlbstatsapi/mlb_api.py:256]: Why do you need a try case here? What search key could possibly cause an exception with a if statement simply checking the string?

Comment (Mattsface) [reply mlbstatsapi/mlb_api.py:256]: If the key isn't present its going to throw a key error

Comment (Mattsface) [reply mlbstatsapi/mlb_api.py:256]: which is why the function was failing in the first place there is a record with no name or fullname from the looks of it

Comment (Mattsface) [reply mlbstatsapi/mlb_api.py:254]: @KCNilssen take a look now

**Review by KCNilssen — APPROVED** [review]

**Pull request merged**